src/incubator/TextField/AccessoryButton.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import Button, {ButtonPropTypes} from '../../components/button';
+
+export default (props: ButtonPropTypes) => {
+  return (
+    <Button link grey10 activeOpacity={props.onPress ? 0.6 : 1} {...props} />
```

Comment:
here you can basically override any of the default configs. Did you mean to give that much flexibility?

Reply:
yes, completely. 
I don't see any reason to block the users from doing anything they want. 
In our private lib we will enforce it to our guidelines.